tests/oauth/test_base.py

```diff
@@ -65,106 +34,124 @@ async def test_oauth(mocker):
 
     app = FastAPI()
 
-    oauth_router = init_oauth_router(**TEST_DATA)
+    # Python doesn't store Classes as variables within a class, rather as a nested class.
+    oauth_router = init_oauth_router(
+        test_information.app_scopes,
+        test_information.user_scopes,
+        test_information.public_domain,
+        test_information.private_key,
+        test_information.app_handle,
+        test_information.api_key,
+        test_information.api_secret_key,
+        OfflineToken,
+        OnlineToken,
+    )
```

Comment:
Python treats class variables that point to other classes as subclasses, even if they are variables. This means I cannot set OfflineToken and OnlineToken in the main `test_information` class, which in turn means I cannot just pass in the object as kwargs.